### ISSUE TITLE
Where is web.config?

### ISSUE THREAD

**Issue by davemm** [MSOpenTech/connectthedots#170]
I am attempting to create the IoT system with my Microsoft band, but can't find the web.config file in the system. Here is the readme that talks about it: connectthedots/Azure/WebSite/WebsitePublish.md

Not being a programmer, I'm not really sure what to do, I'm just figuring this all out as I go, so if someone could explain what I need instead of web.config or where I can find it, I would appreciate it.

**Comment by dinarisio** [comment]
Please use AzurePrep solution's project 'CreateWebConfig' to get ready web.config file, but first you need to initialize cloud tools like Event Hubs using AzurePrep.

**Comment by davemm** [comment]
It is telling me to use createwebconfig.exe, but I don't see that in there. How do I get started using the createwebconfig folder?

**Comment by dinarisio** [comment]
Open AzurePrep.sln in VS, than in Solution Explorer right-click on 'CreateWebConfig' project and select 'Set as StartUp project', then run.

**Comment by davemm** [comment]
Thank you for taking time to help me out. I did as you said. I copeid the newly created web.config over the old one, Then when I tried to publish the website, it gave me the following error: 
Error	
3	Copying file Web.config to obj\Release\TransformWebConfig\original\Web.config failed. Could not find file 'Web.config'.
I thought this would be fixed with the new web.config. I know I've taken a bunch of your time, but if you have some ideas, I'd appreciate it.
